website/docs/docs/collaborate/data-health-signals.md

```diff
@@ -0,0 +1,88 @@
+
+## Data health signal criteria
+
+Each resource has a health state that is determined by specific set of criteria. Select the following tabs to view the criteria for that resource type.
+<Tabs>
+<TabItem value="models" label="Models">
+
+The health state of a model is determined by the following criteria:
+<!-- TODO: remove the 'tbd' lines in the table once meta 4025 is done -->
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ **Healthy**    | All of the following must be true:<br /><br /> - Built successfully in the last run<br />- Built in the last 30 days<br />- Model has tests configured<br />- All tests passed<br />- All upstream sources are fresh or freshness is not applicable (set to `null`)<br />- Has a description |
+| 🟡 **Caution**   | One of the following must be true: <br /><br />- Not built in the last 30 days<br />- Tests are not configured<br />- Tests return warnings<br />- One or more upstream sources are stale:<br />&nbsp;&nbsp;&nbsp;&nbsp;- Has a freshness check configured<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check ran in the past 30 days<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check returned a warning<br />- Missing a description |
+| 🔴 **Degraded**  | One of the following must be true: <br /><br />- Model failed to build<br />- Model has failing tests<br />- One or more upstream sources are stale:<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check hasn’t run in the past 30 days<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check returned an error |
+| ⚪ **Unknown**    | - Unable to determine health of resource; no job runs have processed the resource.         |
+
+</TabItem>
+
+<TabItem value="sources" label="Sources">
+
+The health state of a source is determined by the following criteria:
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ Healthy	| All of the following must be true: <br /><br />- Freshness check configured<br />- Freshness check passed<br />- Freshness check ran in the past 30 days<br />- Has a description |
+| 🟡 Caution	| One of the following must be true: <br /><br />- Freshness check returned a warning<br />- Freshness check not configured<br />- Freshness check not run in the past 30 days<br />- Missing a description |
+| 🔴 Degraded	| - Freshness check returned an error |
+| ⚪ Unknown	| Unable to determine health of resource; no job runs have processed the resource.     |
+
+</TabItem>
+
+<TabItem value="exposures" label="Exposures">
+
+The health state of an exposure is determined by the following criteria:
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ Healthy	| All of the following must be true: <br /><br />- Underlying sources are fresh<br />- Underlying models built successfully<br />- Underlying models’ tests passing<br />-<!-- - Freshness must be applicable <br /> - (TBD) Underlying models built in the last 30 days --> |
```

Comment:
Hiya @mirnawong1 

There's a hyphen after the point "Underlying models’ tests passing" I was unable to see it in the PR to remove it.

Kind Regards
Natalie
<img width="664" alt="Screenshot 2024-12-17 at 11 31 55" src="https://github.com/user-attachments/assets/a5808772-7de4-4412-8478-1a93488bb465" />

Reply:
I don't see this any longer so resolving the comment